lib/native_service/media_permissions_service.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:weforza/native_service/native_service.dart';
+
+/// This class represents an interface for requesting media related permissions.
+abstract base class MediaPermissionsService extends NativeService {
+  const MediaPermissionsService();
+
+  /// Request permission to add photos to the photo gallery of the device.
```

Comment:
Instead of making it specific to iOS, add a fallback to true on other platforms in the implementation.